components/developer/jenkins-core-lts/Makefile

```diff
@@ -87,4 +87,4 @@ install: $(INSTALL_32)
 test: $(NO_TESTS)
 
 # Auto-generated dependencies
-REQUIRED_PACKAGES += runtime/java/openjdk17
+REQUIRED_PACKAGES += runtime/java/openjdk21
```

Comment:
You cannot achieve what you wanted with this change. It is a generated setting (created by running "gmake REQUIRED_PACKAGES"). The system is either using the default openjdk17 at the moment) or something that has been configured explicitly.